packages/maskbook/src/database/global.d.ts

```diff
@@ -1,18 +1,6 @@
 declare module 'idb/with-async-ittr-cjs' {
     export * from 'idb'
 }
-declare module 'metamask-extension-provider' {
-    import type { EventEmitter } from 'events'
-    import type { AbstractProvider } from 'web3-core'
-    export class MetamaskInpageProvider extends EventEmitter implements AbstractProvider {
-        sendAsync(payload: JsonRpcPayload, callback: (error: Error | null, result?: JsonRpcResponse) => void): void
-        send?(payload: JsonRpcPayload, callback: (error: Error | null, result?: JsonRpcResponse) => void): void
-        request?(args: RequestArguments): Promise<any>
-        connected?: boolean
-    }
-    declare function createMetaMaskProvider(): MetamaskInpageProvider
```

Comment:
since `metamask-extension-provider@2.0.0` published, no longer need us to `declare function createMetaMaskProvider`